scripts/percona-release.sh

```diff
@@ -120,11 +120,14 @@ function show_help {
   echo "  Example: $(basename ${0}) enable tools release"
   echo "  Example: $(basename ${0}) enable-only ps-80 experimental"
   echo "  Example: $(basename ${0}) setup ps57"
+  echo "  Example: $(basename ${0}) setup -y ps57"
+  echo "  Example: $(basename ${0}) setup ps57 -y"
   echo
   echo "-> Available commands:       ${COMMANDS}"
   echo "-> Available setup products: ${ALIASES}"
   echo "-> Available repositories:   ${REPOSITORIES}"
   echo "-> Available components:     ${COMPONENTS}"
+  echo "=> Commands which have interactive questions e.g \"setup\" could be automatically accepted by setting option \"-y\""
```

Comment:
I'm sorry I can't understand what you mean.  
AFAIU by running the \"setup\" command with the \"-y\" option, the user automatically confirms all actions this command may require. Correct?

Reply:
Yeah, only setup command asks for confirmation when running on CentOS 8
So it can be accepted without interactive shell where user can put Y | N

Used as example [this](http://man7.org/linux/man-pages/man8/yum.8.html) doc:
```
       -y, --assumeyes
              Automatically answer yes for all questions.
```

Reply:
Is setup the only command with interactive shell? Does it behave the same on other distributions, e.g. Ubuntu?
If the answer is yes for both i'd word it as follows:
"The  "-y" option for the setup command automatically answers "yes" for all interactive questions."

Reply:
ATM, it's the only one command which has such feature and only on CentOS

> If the answer is yes for both i'd word it as follows:
"The "-y" option for the setup command automatically answers "yes" for all interactive questions."

Agreed, thank you

Reply:
It only affects CentOS but I've added your variant which goes for both, I think there's no need to point _that_ clear for users